<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1195</width>
    <height>725</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1195</width>
    <height>725</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1195</width>
    <height>725</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>华文行楷</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/bkg.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exit_pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/button.png);
color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/ck.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>501</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/xx.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="save_pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/button.png);
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>保存记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/button.png);
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QLabel" name="tem_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>500</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffff99;</string>
    </property>
    <property name="text">
     <string>  温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="safe_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff3300;</string>
    </property>
    <property name="text">
     <string>   电量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="hum_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>600</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffff99;</string>
    </property>
    <property name="text">
     <string>  湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="light_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffff99;</string>
    </property>
    <property name="text">
     <string>  光照：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_off_pushButton_m">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/led.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="smg_on_pushButton_m">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>560</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/smg_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="wind_off_pushButton_m">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/wind.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alarm_on_pushButton_m">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>560</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/alarm_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="smg_off_pushButton_m">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>560</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/smg.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="light_on_pushButton_m">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/led_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="wind_on_pushButton_m">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/wind_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alarm_off_pushButton_m">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>560</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/alarm.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="bat_textBrowser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>610</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/tembk.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="light_textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/tembk.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="temp_textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/tembk.png);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="hum_textBrowser">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>600</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/tembk.png);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="help_groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>271</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/print.png);</string>
    </property>
    <property name="title">
     <string>帮助信息</string>
    </property>
    <widget class="QTextBrowser" name="help_textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/print.png);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;1、该软件通过Zigbee与M0进行无线通信。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; color:#5500ff;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;2、该软件可直观实时的显示M0节点所处环境的温度（℃）、湿度（RH）、光照（Lax） 以及当前M0的电量（V）状态。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; color:#5500ff;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;3、控制组按钮可以对M0外围设备进行实时控制。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; color:#5500ff;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;4、保存记录功能，通过截屏，可以随时保存当前页面信息。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt; color:#5500ff;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#5500ff;&quot;&gt;5、该软件版权由刀塔团队所有，如需源码，发邮件到：yiranshiwx@sina.com&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="know_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>292</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/tembk.png);
background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>知道了</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
